### PR TITLE
Created HomePage

### PR DESCRIPTION
**Pull request by ericta-portfolio**

### CODE REVIEW

File: frontend/src/HomePage/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Testimonial from "./Testimonial";
+import testimonials from "./testimonials";
+
+export default function Homepage() {
+  function createTestimonial(testimonial) {
+    return (
```
Comment (jpnoe) [inline frontend/src/HomePage/Homepage.jsx:7]: good idea to reuse this code

**Review by jpnoe — APPROVED** [review]
everything looks great!

**Review by lalvessiqueira — COMMENTED** [review]
Looks good! Still want to see how it looks when we run it!

**Review by lalvessiqueira — APPROVED** [review]

File: frontend/src/HomePage/Homepage.jsx (hunk 1 above)
Comment (pvolnuhi) [reply frontend/src/HomePage/Homepage.jsx:7]: looks good, and reused efficiently

**Pull request merged**